### PR TITLE
update Debian package requirements

### PR DESCRIPTION
**Pull request by gemesa**
`pkg-config` `libcurl4-openssl-dev` `libssl-dev` are not necessary anymore (since https://github.com/ZerBea/hcxdumptool/commit/d1f23265349bae552659e964cfb7f2953548d93a and https://github.com/ZerBea/hcxdumptool/commit/311e09325d70a0774d25f687b285c3af26ca7519), `zlib1g-dev` has never been required, I think it was just copied from `hcxtools`, so lets remove them.

### CODE REVIEW

**Comment by ZerBea** [comment]
Thanks.
I'm not sure if refacterd hcxdumptool will find its way to Bookworm (which is in freeze), but good to have a notice here.
I haven't removed this, because I thought it is possible to add this features to hcxdumptool, too.
Unfortunately it doesn't work as expected and I dropped it.
From the lessons learned:
It is not feasible  to recover PSKs on the fly (e.g like aircrack-ng wep).
It is not feasible to gzip the dum file on the fly.
It is not feasible to upload the pcapng file to wpa-sec on the fly.
It is not feasible to hold a BT or WLAN or SSH connection while running hcxdumptool.

Because hcxdumptool is an interactive tool that must(!) respond to a target within a precisely defined time.
Regarding features, we have a ranking. This ranking must be strictly observed:
time response behavior
cpu cycles of a Raspberry Pi

In other words:
form follows function
keep it small and simple
do one thing and do it well
